src/utils/notification-template.ts

```diff
@@ -43,11 +44,11 @@ export const render = (transport: TransportName, notification: Notification, var
   const path = join(BASE_TEMPLATES_PATH, getTemplateFilePath(transport, notification.type, variant));
   return ejs.renderFile(
```

Comment:
Ejs? I didn't know we were using templating :D
That's cool

Reply:
@wadader As this repo is meant to be installable by our users, we opted for `ejs` to allow them to customize the Slack/Telegram/Email notifications. We considered SendGrid templates for emails, but that would force them to use an specific third-party solution.